Documenting the custom field delete command (#352)
mautic/user-documentation: docs/contacts/custom_fields.rst

If you configure this option, the new Custom Field becomes visible in the list of Custom Fields. The Custom Field remains unpublished until you run the command ``bin/console mautic:custom-field:create-column``. This command creates the actual column in the table and publishes the field metadata.

> Similarly, the ``bin/console mautic:custom-field:delete-column`` command deletes the actual column in the table if the ``create_custom_field_in_background`` config option is set to true. The column gets soft-deleted and removed from the user interface. But the data is still present in the database until you run the command to delete the column.

Reviewer: ```suggestion
Similarly, the ``bin/console mautic:custom-field:delete-column`` command deletes the actual column in the table if you have turned on the ``create_custom_field_in_background`` config option. The column gets soft-deleted and removed from the user interface, but the data is still present in the database until you run the command to delete the column.
```

Passive -> Active voice
